.github/workflows/workflow.yaml

```diff
@@ -317,6 +317,60 @@ jobs:
             done
           done
 
+  benchmark:
+    name: "Benchmark"
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+
+      - name: Download release benchmark file
+        uses: robinraju/release-downloader@v1.6
+        id: get-latest-benchmark
+        continue-on-error: true
```

Comment:
also could we make this wf fail actually? what were the arguments for this again?

I don't want us to be tricked by the ✅ in the checks summary only to find out that we can't actually download the data that we need. 

https://github.com/open-policy-agent/gatekeeper/actions/runs/3680438765/jobs/6226037201

At least an ❌  signals to the author to investigate. Wdyt?

Reply:
The argument was that just missing release benchmark data is not enough reason to fail the workflow. But, yeah I think somehow outputting the error data or indicating that there was an error please look at the result of the job would be useful. I think we could post the message on the PR the same way we are outputting benchstat data to indicate the error on retrieving past data.